server/app/com/xsn/explorer/services/synchronizer/repository/MasternodeRepository.scala

```diff
@@ -0,0 +1,32 @@
+package com.xsn.explorer.services.synchronizer.repository
+
+import akka.pattern.ask
+import akka.util.Timeout
+import com.xsn.explorer.models.rpc.Masternode
+import com.xsn.explorer.services.synchronizer.MasternodeSynchronizerActor
+import javax.inject.Inject
+import scala.concurrent.duration._
+import com.xsn.explorer.models.values.IPAddress
+import scala.concurrent.Future
+
+trait MasternodeRepository {
+  def get(): Future[List[Masternode]]
+  def getOne(ipAddress: IPAddress): Future[Option[Masternode]]
+}
+
+object MasternodeRepository {
+
+  class ActorImpl @Inject()(actor: MasternodeSynchronizerActor.Ref) extends MasternodeRepository {
+    override def get(): Future[List[Masternode]] = {
+      implicit val timeout: Timeout = 10.seconds
```

Comment:
Define this timeout as a private value outisde of the methods, so that it's shared by both of them.